css/style.css

```diff
@@ -1096,7 +1096,7 @@ body.mobile-nav-active #mobile-nav-toggle {
   border-bottom: 1px solid #e0e5fa;*/
   overflow: hidden;
   /*background: rgba(255, 255, 255, 0.5);*/
-  height: 160px;
+  /* height: 160px; */
```

Comment:
Is it on purpose to comment it? If yes, why not remove it?

Reply:
Most of our changes were commented,  and I forgot to remove it when I was satisfied. This will be all resolved once we move to a static site generator

Reply:
👌